data/bedrock/common/features.json

```diff
@@ -5,8 +5,8 @@
     "versions": ["1.16_major", "latest"]
   },
   {
-    "name": "serializedItemNbtIsExtraData",
-    "description": "in newer versions, NBT fields are within the extra field, but before 1.16.220, they're not",
+    "name": "itemMetadataAndCountFields",
```

Comment:
I think `itemSerializeUsesAuxValue` or `itemSerializeCombinesMetadataAndCountField` would be better names (the former being less long)

Reply:
Yes, that's a lot better (I just had the usual *can't think of a good name*)